flake.nix

```diff
@@ -0,0 +1,24 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

Comment:
@scientiac Is there any reason (e.g. new feature, fix) for using `unstable`?

Reply:
NixOS follows both release models (kinda) The normal release model like Ubuntu (eg: 24.04, 24.11 etc.) and the rolling release model (Like arch) so, the unstable is basically a rolling release package repo (this is also where the python packages and other dependencies are pulled from) to ensure the latest version of dependencies.